README.md

```diff
@@ -1,75 +1,30 @@
-    │       |                                       If AICoE CI GitHub App (https://github.com/apps/aicoe-ci) is installed, once the pull request is merged a new tag is created by the bot
-    │       |                                       and the pipeline to build and push image starts.
-    │       |
-    |       └── redeliver_container_image.md    <- If the tag exists and AICoE CI GitHub App (https://github.com/apps/aicoe-ci) is installed, the issue will retrigger the pipeline to build and
-    │                                               push image container image. NOTE: It should be used if the pipeline triggered with the {major|minor|patch}_release failed for any reason.
-    |
-    └── tox.ini                                 <- tox file with settings for running tox; see tox.readthedocs.io
+## Time Series Forecasting Models
 
---------
+Time series forecasting models are statistical models that use historical data to predict future values. These models are used in a variety of industries, including finance, retail, and manufacturing.
 
-<p><small>Project based on the <a target="_blank" href="https://drivendata.github.io/cookiecutter-data-science/">cookiecutter data science project template</a>. #cookiecutterdatascience</small></p>
+There are many different types of time series forecasting models, but some of the most common include:
+
+* Autoregressive models (AR): These models use past values of the time series to predict future values.
+* Moving average models (MA): These models use past errors in the time series to predict future values.
+* Autoregressive moving average models (ARMA): These models combine AR and MA models.
+* Seasonal autoregressive integrated moving average models (SARIMA): These models are used for time series that have seasonal patterns.
+
+The choice of which time series forecasting model to use depends on the characteristics of the time series data. For example, if the time series data is very noisy, then a model that is robust to noise, such as an ARMA model, may be a good choice. For this project, we have chosen the ARIMA model.
+
+## Data
+
+To develop these open source models, we are using a public dataset from [Kaggle](https://www.kaggle.com/datasets/census/business-and-industry-reports?select=notes.txt). The data consists of unique account IDs and monthly revenue which we primarily use to train our model. We would also like to use Generative AI techniques to generate public datasets that mimic internal company/business customer revenue data, you can follow [this issue](https://github.com/redhat-et/customer-growth-model/issues/7) to learn more about our progress.
+
+## Repo Structure
+
+* `data`
+    * `data/raw` - directory contains the raw dataset (obtained from Kaggle)
+    * `data/processed` - directory contains the model predictions and classification output files
+    
+* `notebooks` - Jupyter notebooks developed for exploratory analysis, model training and model classification.
```

Comment:
Shall we provide the hyperlink to the "data/raw", "data/processed", and "notebook" folder? Wdyt?

Reply:
thanks for the catch, will add it 👍